setup.ts

```diff
@@ -922,12 +907,16 @@ async function main(): Promise<void> {
     await recaptchaSiteKey();
   }
 
+  console.log(
+    "\n You can configure either SMTP or Mail for sending emails through Talawa.\n"
```

Comment:
1. Should be `You can configure SMTP for sending emails through Talawa`
2. `SMTP` is a mail sending protocol

Reply:
Never mind. This can be resolved for the reasons stated previously